src/test/java/net/imagej/ops/AbstractOpTest.java

```diff
@@ -235,6 +238,10 @@ private int pseudoRandom() {
 		}
 		assertFalse("More elements than expected", a.hasNext());
 	}
+	
+	public static <T> RandomAccessible<T> deinterval(RandomAccessibleInterval<T> input){
+		return Views.extendBorder(input);		
```

Comment:
Looking at where this method is used, replacing the original casts with calls to it seems legit. In general, however, this is not the case: with L243 you are replacing an already existing `OutOfBoundsFactory` of input with an `OutOfBoundsBorderFactory`.